include/wasm/WasmModule.h

```diff
@@ -23,13 +23,15 @@
 // Note: this is *not* controlling the size provisioned by the linker, that is
 // hard-coded in the build. This variable is just here for reference and must be
 // updated to match the value in the build.
-#define STACK_SIZE 4 * ONE_MB_BYTES
+#define STACK_SIZE (4 * ONE_MB_BYTES)
+#define THREAD_STACK_SIZE (2 * ONE_MB_BYTES)
+#define THREAD_STACK_POOL_SIZE 20
```

Comment:
Threads are now handled at the generic `WasmModule` level, rather than only in the `WAVMWasmModule`